src/GraphQL/Validation/Errors.Custom/InputFieldsAndArgumentsOfCorrectLengthError.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using GraphQL.Language.AST;
+
+namespace GraphQL.Validation.Errors
+{
+    /// <inheritdoc cref="Rules.InputFieldsAndArgumentsOfCorrectLength"/>
+    [Serializable]
+    public class InputFieldsAndArgumentsOfCorrectLengthError : ValidationError
```

Comment:
Would not include in library.

Reply:
Let's think about the name of a new project to put this into.

Reply:
What will we decide?

Reply:
No opinion.  If we have worked out all the "bugs", then I'd probably leave it in the main project for now.  It may be useful, and at least it serves as an excellent sample.

Reply:
Then I will keep it here for now.